piqueserver/server.py

```diff
@@ -65,13 +66,25 @@
 except NameError:
     pass
 
-# default passwords hardcoded in config
-DEFAULT_PASSWORDS = {
-    'admin': ['adminpass1', 'adminpass2', 'adminpass3'],
-    'moderator': ['modpass'],
-    'guard': ['guardpass'],
-    'trusted': ['trustedpass']
-}
+def check_passwords(passwords):
```

Comment:
Note for future: Once we have TOML, we can just comment out the passwords section by default